src/routes/users.route.js

```diff
@@ -97,4 +98,33 @@ userRouter.get(
   getReadingStats,
 );
 
+/**
+ * @swagger
+ *
+ * /api/v1/users/profiles/:username/stats:
```

Comment:
your documentation is showing for `stats`

Reply:
Thank you, this has been resolved